content/en/blog/memory_management/index.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Memory management techniques"
+date: 2022-05-20T08:00:00Z
+draft: false
+contributors: [Cloud Hypervisor Team]
+---
+
+Depending on the expectations around the workload running in a Virtual Machine
+(VM), as well as the agreement between a customer and a Cloud Service Provider
+(CSP), multiple use cases related to memory management between host and guest
+arise and they can be addressed through different techniques.
+
+### Host managed
+
+This section is about how memory can be managed by the host, as it knows the VM
+is going to need more or less memory during its runtime.
+
+This is particularly well illustrated by the Cloud Native use case, involving
+containers to be created or destroyed, leading to the need to adjust the amount
+of memory available.
+
+We can also take the example of a user running a single VM with a certain amount
+of RAM until he needs more because of a different type of workload that requires
+a larger amount of memory.
+
+For both of these use cases, the goal is to save resources and therefore money.

```

Comment:
It might be presumptuous to make the connection between resources and money so that could be omitted